community/planning/consensus_runner.md

```diff
@@ -0,0 +1,281 @@
+---
+tags: [Mermaid]
```

Comment:
This header shouldn't be in the file unless mermaid is used.

Reply:
Removed